packages/loader/index.js

```diff
@@ -1,15 +1,27 @@
 const { getOptions } = require('loader-utils')
 const mdx = require('@mdx-js/mdx')
 
+function getImportCode(options) {
```

Comment:
I wonder if we should have a separate loader specifically for Vue? I know it'd be a decent amount of duplication, but  I typically prefer to keep loaders free from as much logic as possible.

Reply:
I agree on this. A loader scope should be limited to one action, a option could fine tune the transpilation but not changing it scope.